source/prerequisites/my-sql.txt

```diff
@@ -10,4 +10,171 @@ MySQL Migration Prerequisites
+
+         .. step:: Locate and update the MySQL configuration file
+
+            a. Run the following SQL query to get the ``server_id`` 
+               value for your MySQL instance:
+
+                  .. tabs::
+
+                     .. tab:: MySql 8.x
+                        :tabid: mysql-8x
+
+                        .. code-block:: sql
+                           :copyable: true
+
+                           SELECT variable_value 
+                           FROM 
+                           performance_schema.global_variables 
+                           WHERE variable_name='server_id';
+
+                     .. tab:: MySql 5.x
+                        :tabid: mysql-5x
+
+                        .. code-block:: sql
+                           :copyable: true
+
+                           SELECT variable_value 
+                           FROM 
+                           information_schema.global_variables 
+                           WHERE variable_name='server_id';
+
+            #. Locate the config file for your MySQL instance by running 
+               the following ``mysqld`` command in your terminal:
+
+               .. tabs::
+
+                  .. tab:: Windows
+                     :tabid: windows
+
+                     .. code-block:: 
+                        :copyable: true
+
+                        mysql --help | findstr cnf
+
+                  .. tab:: MacOS / Linux
+                     :tabid: macos-linux
+
+                     .. code-block:: 
+                        :copyable: true
+
+                        mysql --help | grep cnf
+
+            #. Under the ``[mysqld]`` section of your MySQL 
+               configuration file add the following lines. Replace 
+               the ``XXXXX`` value with the ``server_id`` from the 
+               previous query:
+
+               .. tabs::
+
+                  .. tab:: MySql 8.x
+                     :tabid: mysql-8x
+
+                     .. code-block::
+                        :copyable: true
+
+                        server-id = XXXXX
+                        log_bin = mysql-bin
+                        binlog_format = ROW
+                        binlog_row_image = FULL
+                        binlog_expire_logs_seconds = 864000
+
+                  .. tab:: MySql 5.x
+                     :tabid: mysql-5x
+
+                     .. code-block:: 
+                        :copyable: true
+
+                        server-id = XXXXX
+                        log_bin = mysql-bin
+                        binlog_format = ROW
+                        binlog_row_image = FULL
+                        expire_log_days = 10 
+
+               .. note::
+
+                  If your running MySQL on AWS RDS, you must enable 
+                  automated backups for binary logging. If automated 
+                  backups are not enabled, ``Binlog`` is disabled even
+                  if setting the values in the configuration file.
```

Comment:
[Question]

Sorry, I am just a little confused with the ending here. Is it disabled even if you set the values in the config file?

Reply:
This is for the [AWS RDS ](https://aws.amazon.com/rds/) version of MySQL, a cloud hosted (non bare metal install). This is supplemental information for only those people with MySQL AWS instances. And yes the point here is that even if you make the configuration changes in the file, if automated backups are not enabled, then it will not work.